### ISSUE TITLE
druid migration issue

### ISSUE THREAD

**Issue by pdngayan** [apache/druid#8730]
I am using druid 0.13 as single server installation. And now we want update
in to 0.16 as cluster deployment. We are having issue when migration the
deep storage in Azure blob storage. Druid documentations are not clear
about this.Can anyone help us..?

**Comment by jihoonson** [comment]
Hi @pdngayan, have you had a chance to check https://druid.apache.org/docs/latest/operations/deep-storage-migration.html? If so, what part was not clear to you? Please feel free to ask more questions if you have. Also, you may want to check what has changed between 0.13 and 0.16 and what you should be aware of for upgrading your cluster. Please check the release notes for those: https://github.com/apache/incubator-druid/releases.

**Comment by pdngayan** [comment]
Hi,

Now we are running druid 0.13.0-incubating as single-server deployments.
Our goal is to migrate current setup in to  druid Cluster setup ( V :
0.16.0-incubating)

First we try to migrate 0.13.0 to 0.16.0 single-server and there is a
problem when migrating deep-storage because the structure has been changed
from 0.13 to 0.16. ( In current version 0.13.0,  there are two files in
segments  *descriptor.json & index.zip  *But in new version 0.16.0
*descriptor.json* is not there ). Also when we copy the segments in to new
location still it is searching for old path.

Also we used Export Metadata Tool to migrate  durby metadate in to
Postgress in clusterd environment and it was success.
But We were unable to find  a way to migrate deep storage in to clustered
environment.  We are using Azure blob storage as deep storage and your
documentation only provide some
 details on migrate to S3 but still not clear. ( means there is no other
tool than export metadata tool)

So we would like to know what is the best way to follow to migrate data
from druid 0.13.0-incubating single-server deployment to druid
0.16.0-incubating Clustered deployment.

Thanks
Gayan

**Comment by pdngayan** [comment]
Hi Jad Naous,

Now we are running druid 0.13.0-incubating as single-server deployments.
Our goal is to migrate current setup in to  druid Cluster setup ( V :
0.16.0-incubating)

First we try to migrate 0.13.0 to 0.16.0 single-server and there is a
problem when migrating deep-storage because the structure has been changed
from 0.13 to 0.16. ( In current version 0.13.0,  there are two files in
segments  *descriptor.json & index.zip  *But in new version 0.16.0
*descriptor.json* is not there ). Also when we copy the segments in to new
location still it is searching for old path.

Also we used Export Metadata Tool to migrate  durby metadate in to
Postgress in clusterd environment and it was success.
But We were unable to find  a way to migrate deep storage in to clustered
environment.  We are using Azure blob storage as deep storage and your
documentation only provide some
 details on migrate to S3 but still not clear. ( means there is no other
tool than export metadata tool)

So we would like to know what is the best way to follow to migrate data
from druid 0.13.0-incubating single-server deployment to druid
0.16.0-incubating Clustered deployment.

Thanks
Gayan
